applications/openshift/scc/scc_limit_container_allowed_capabilities/rule.yml

```diff
@@ -38,3 +38,18 @@ ocil: |-
     Review each SCC and determine that only required capabilities are either
     completely added as a list entry under <tt>allowedCapabilities</tt>,
     or that all the un-required capabilities are dropped for containers and SCCs.
+
+warnings:
+- general: |-
+    {{{ openshift_filtered_cluster_setting({'/apis/security.openshift.io/v1/securitycontextconstraints': '[.items[] | select(.metadata.name != "privileged")] | map(.allowedCapabilities == null)'}) | indent(4) }}}
```

Comment:
With the developer docs section for this feature, it would be good to have a link to a reference for jq, or some examples of common queries.